File: Chapter 6/src/EvenNums.cpp
#include<iostream>

using namespace std;

int main(){
 int number;
 int LIMIT = 100;
 for(number = 2; number <= LIMIT; number++){
 cout<<number<<endl;
 if((number % 20)==0){
 cout<<"\n"<<endl;
 }
 }
}

File: Chapter 6/src/EvenNums.java
public class EvenNums {
    public static void main(String[] args){
      int number;
      int NEW_LINE = 20;
      final int LIMIT = 100;

      for(number = 2; number <= LIMIT ; number++){
          System.out.println(number);
          if((number % NEW_LINE)==0)
          System.out.println("\n");
      }
    }
}
